drivers/gpio/gpio_rv32m1.c

```diff
@@ -97,44 +149,27 @@ static int gpio_rv32m1_configure(struct device *dev,
 	 */
 	mask |= PORT_PCR_IRQC_MASK;
```

Comment:
Should be removed?

Reply:
No, it is still needed

Reply:
The code on line 162 will clear IRQ configuration from PCR register but we don't configure interrupts here. If we merge #19553 the `gpio_rv32m1_configure` would still modify interrupt configuration.